draft-davidson-pp-protocol.md

```diff
@@ -203,7 +248,8 @@ client.
 The client initialises a global storage system `store` that allows it
 store the tokens that are received during issuance. The storage system
 is a map of Issuer identifiers (`Issuer.id`) to arrays of stored tokens.
-We assume that the client knows the Issuer public key `pkI` ahead of time.
+We assume that the client knows the Issuer public key `pkI` ahead of
```

Comment:
And of course that clients have a mechanism to verify their view of the Issuer public key is the same as all other clients.

Reply:
Added a link to the discussion about this in the architecture doc.